### PR TITLE
Removed Nexus Plugin

### PR DESCRIPTION
**Pull request by gsvic**

### CODE REVIEW

File: ci/tiledb-java-final-jar.yml [hunk 1]
```diff
@@ -71,4 +71,12 @@ steps:
       GPG_SECRET_KEYS_ENC: $(GPG_SECRET_KEYS_ENC)
       SONATYPE_USERNAME: $(SONATYPE_USERNAME)
       SONATYPE_PASSWORD: $(SONATYPE_PASSWORD)
-    displayName: "Publish"
\ No newline at end of file
+    displayName: "Publish"
+
+  - task: GithubRelease@0
```
Comment (Shelnutt2) [inline ci/tiledb-java-final-jar.yml:76]: Let's do this in a separate PR, that way the build.grade change is clean.

Comment (gsvic) [reply ci/tiledb-java-final-jar.yml]: done
